src/PrestaShopBundle/Resources/config/services/bundle/services.yml

```diff
@@ -61,9 +61,7 @@ services:
     class: PrestaShop\PrestaShop\Adapter\Hook\HookDispatcher
     arguments:
       - '@request_stack'
-    calls:
-      - [ addSubscriber, [ "@prestashop.adapter.legacy.hook.subscriber" ] ]
-      - [ addSubscriber, [ "@prestashop.adapter.legacy.block.helper.subscriber" ] ]
+      - !tagged core.legacy.hook.subscriber
```

Comment:
can you add a `prestashop.` prefix on your tag name ?

Reply:
We have another tags beginning with "core", I think we have to keep a coherence ? ex : core.product_command_builder